dev-resources/logback.xml

```diff
@@ -5,11 +5,11 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
+    <logger name="puppetlabs" level="trace"/>
```

Comment:
don't think we want this change.  It results in lots of messages being logged during `lein test`, which we generally try to avoid (although we often fail).  This also shouldn't affect any logging tests or anything like that ... not sure if there's some other reason it would be necessary?